toolkit/tools/pkg/imagecustomizerlib/customizepartitionsfilecopy.go

```diff
@@ -21,14 +21,20 @@ func customizePartitionsUsingFileCopy(buildDir string, baseConfigPath string, co
 	}
 	defer existingImageConnection.Close()
 
+	existingSELinuxMode, err := getCurrentSELinuxMode(existingImageConnection.Chroot())
```

Comment:
Nit-pick just for the sake of consistency: since we called the function `getCurrentSELinuxMode`, it might be worth to stick to this naming for the variable as well.
```suggestion
	currentSELinuxMode, err := getCurrentSELinuxMode(existingImageConnection.Chroot())
```

Or swap it the other way around.:)

Reply:
fixed